src/Backend/WASMWrapper.js

```diff
@@ -82,6 +107,17 @@ export default class WASMWrapper {
         return this.getNames(untyped, 'dedx_get_material_name')
     }
 
+    /**
+     * Creates a dataseries using data for a given program ion and material
+     * Chooses appropriate method of generation based on method and isLog params
+     * @param {LibdedxEntity} program - a libdedx program object
+     * @param {LibdedxEntity} ion - a libdedx ion object
+     * @param {LibdedxEntity} material - a libdedx material object
+     * @param {number} method - data series generation method
+     * @param {number} plotUsing - number of points to generate the plot for
```

Comment:
Rename `plotUsing` to somethin that is related to number of points, like `noOfPoints`